accepted/0000-add-preunpack-life-cycle-script.md

```diff
@@ -0,0 +1,73 @@
+# RFC: Add `preunpack` life cycle script
+
+## Summary
+
+Add a `preunpack` life cycle script that runs before anything else.
```

Comment:
Hopefully this wouldn’t repeat the mistake of many other lifecycle scripts - if it’s named “prefoo”, the only thing it should run immediately preceding is “foo”. Thus, “preunpack” only runs before unpack, not before “anything else”, and nothing should run between preunpack and unpack.

Reply:
Agreed, I'll update the wording on this.